### ISSUE TITLE
Triangle before proof not correctly shown if `showmore` is enabled

### ISSUE THREAD

**Issue by utensil** [PatrickMassot/leanblueprint#8]
## TL;DR

In project PFR and alike, the triangle before a proof shows ▼/▶ for proof collapsing/expansion, which is the opposite of how one expects them, where ▼ should indicate proof expansion. Also, it's the opposite of project LTE.

Reported by @kbuzzard .

### What's expected

In LTE ( [code](https://github.com/leanprover-community/liquid/tree/master), [rendered](https://leanprover-community.github.io/liquid/) ), `showmore` is NOT enabled, the triangle before proof is shown as ▼, and the proof is expanded by default, so ▼ indicates proof expansion correctly.

Afterward, clicking on the triangle works as expected, the triangle toggles between ▼ and ▶ , indicating the proof is expanded and collapsed, respectively.

<img width="644" alt="image" src="https://github.com/PatrickMassot/leanblueprint/assets/64258/2fd527c5-47f8-46b5-b66f-706ba0d9ed18">

### What went wrong

In PFR ( [code](https://github.com/teorth/pfr), [rendered](https://teorth.github.io/pfr/blueprint/) ), `showmore` is enabled, besides the eye +/- icons appear below and can control the level of details to show, by default, it collapses proofs, but the triangle before proof is still shown as ▼, so ▼ is not correctly corresponding to the proof collapsing status but the opposite.

<img width="738" alt="image" src="https://github.com/PatrickMassot/leanblueprint/assets/64258/0d14c7fb-0a50-4c8c-8577-2e92f9c02d26">

Afterward, clicking on the triangle will never works as expected, it's always the opposite, unless one click on the eye +/- icon odd times, then this would be reversed and fixed.

## Cause

The clicking of the triangle and the `show more` handles the triangle states separately, causing the confliction, plus they have different defaults.

## Workaround

Don't enable `showmore` until this issue is fixed: In `web.tex`

Change the line like 

```
\usepackage[showmore, dep_graph, project=../../]{blueprint}
```
to 

```
\usepackage[dep_graph, project=../../]{blueprint}
```

**Comment by PatrickMassot** [comment]
Thanks for the report. I noticed something was weird but never bothered because this convention always seemed unclear to me. It should be fixed by [d336038](https://github.com/PatrickMassot/leanblueprint/commit/d3360380ba42c48c7a2f30b17c03226618ab080f).
